source/configuration_manual/howto/director_with_lua.rst

```diff
@@ -0,0 +1,115 @@
+.. _howto_director_lua:
+
+==============================
+Run director with Lua on proxy
+==============================
+
+
+.. note:: Lua director is not supported for Dovecot Pro.
+
+Small scale deployments using NFS or other shared storage need a way to ensure users
+end up in same backend. For this purpose, a simple Lua script can be used.
+
+The script can be run on a proxy directly, or one can have one dedicated proxy node acting
+as a Lua director node running the script.
+
+Prerequisites
+-------------
+
+This script can use any SQL driver supported by Lua DBI.
+If you want to use SQLite, or some other file based database, you can only run this on single node.
+
+You will need `Lua DBI <https://github.com/mwild1/luadbi>`_, and a database driver for it.
+For the shard version, you will need to install crc32 library.
```

Comment:
Multiple nitpicks:
- "[...] you can only run this on _a_ single node.",
- "[...] you will need to install _the_ crc32 library." <- I don't know lua, but if that is an external package, this might also benefit from a library link,
- I feel like the first sentence of both the first and second paragraph should be combined, as otherwise you mention the used library before clearly stating where this comes from. Maybe something like: "This script requires <link to Lua DBI>, which makes it suitable for any supported SQL driver." Or something like this.
- Additionally - as a layman - I would appreciate a short note, what the "shard version" refers to.